modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -47,6 +48,7 @@
 /**
  * @author Feliphe Marinho
  */
+@FeatureFlags("LPS-187854")
```

Comment:
Could you separate the add FF in another commit, please?

Reply:
I leave it up to you to decide this :+1: